lms/djangoapps/verify_student/urls.py

```diff
@@ -139,4 +139,9 @@
         views.submit_photos_for_verification,
         name="verify_student_submit_photos"
     ),
+    url(
+        r'^reverify/{course_id}/'.format(course_id=settings.COURSE_ID_PATTERN) + '(?P<checkpoint_name>\w+)/$',
```

Comment:
Nitpick: please format the full string rather than using `+`:
```
r'^reverify/{course_id}/(?P<checkpoint_name>\w+)/$'.format(course_id=settings.COURSE_ID_PATTERN)
```